common/gpu/intel/default.nix

```diff
@@ -1,15 +1,26 @@
 { config, lib, pkgs, ... }:
 
 {
-  boot.initrd.kernelModules = [ "i915" ];
-
-  environment.variables = {
-    VDPAU_DRIVER = lib.mkIf config.hardware.opengl.enable (lib.mkDefault "va_gl");
+  options.hardware.intelgpu.loadInInitrd = lib.mkEnableOption (lib.mdDoc
```

Comment:
Is there a value of making this an option? People will probably not discover it.

Reply:
Saw it was done like this in the amd-module and encouraged in some issues to use options - I don't mind just removing it from framework but maybe not everyone is happy - this way it keeps backwards compatible...

Reply:
OK